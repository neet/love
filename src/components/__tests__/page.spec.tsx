import * as React from 'react';
import * as ReactTestRenderer from 'react-test-renderer';
import Page from '../page';

describe('Page', () => {
  it('renders a page with given title and excerpt', () => {
    const tree = ReactTestRenderer.create(
      <Page
        title='Awesome article'
        excerpt='This article is awesome you should read'
      >
        <p>This is a article</p>
      </Page>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
