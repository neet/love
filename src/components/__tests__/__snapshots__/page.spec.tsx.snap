// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders a page with given title and excerpt 1`] = `
<article
  className="page"
>
  <header
    className="page__header"
    role="banner"
  >
    <div
      className="page__hgroup"
    >
      <h1
        className="page__title"
      >
        Awesome article
      </h1>
      <span
        className="page__excerpt"
      >
        This article is awesome you should read
      </span>
    </div>
  </header>
  <hr
    aria-hidden={true}
    className="page__separetar"
  />
  <p>
    This is a article
  </p>
</article>
`;
